bin/ch/stdafx.h

```diff
@@ -268,7 +267,7 @@ class AutoString
 
     size_t GetLength()
```

Comment:
`AutoString::GetLength` is used only in `Debugger.cpp` (namely, https://github.com/Microsoft/ChakraCore/blob/master/bin/ch/Debugger.cpp#L497). If ChakraFull doesn't have any additional dependencies we could do away with `AutoString::GetLength`

Reply:
@Cellule what do you think?